controller/work_item_link_type.go

```diff
@@ -89,8 +91,7 @@ func enrichLinkTypeList(ctx *workItemLinkContext, list *app.WorkItemLinkTypeList
 	for categoryID := range categoryIDMap {
 		modelCategory, err := ctx.Application.WorkItemLinkCategories().Load(ctx.Context, categoryID)
 		if err != nil {
-			jerrors, httpStatusCode := jsonapi.ErrorToJSONAPIErrors(err)
-			return ctx.ResponseData.Service.Send(ctx.Context, httpStatusCode, jerrors)
+			err
```

Comment:
the reason for such a change is that this block of code belongs to the `enrichLinkTypeList` utility function. If it fails, I think that it should just return the `err`. Then the calling controller method will handle the `err` and return the appropriate HTTP response:
````go
err = enrichLinkTypeList(linkCtx, &appLinkTypes)
if err != nil {
	return jsonapi.JSONErrorResponse(ctx, goa.ErrInternal("Failed to enrich link types: %s", err.Error()))
}
````

isn't it much simpler this way ?